netlify.toml

```diff
@@ -9,6 +9,7 @@
   REACT_APP_BLOCK_EXPLORER="https://www.mintscan.io"
   REACT_APP_LEDGER_CHAIN_ID = "regen-1"
   REACT_APP_SENTRY_ENVIRONMENT="production"
+  REACT_APP_ANALYTICS_ENABLED="true"
```

Comment:
i added this flag only to the production netlify context. this is so that we only send analytics data from production. in the future we could implement it in such a way where we can still send data from the development environment, which may be needed for further tracking development work.

what is left to do to support a development environment for analytics work is to create a development version of the GA4 setup and/or a development version of the measurement id, which is being hardcoded right now in the analytics setup code.

once that is done, we can make use of an environment variable for the measurement id. and since we already have a development environment for amplitude, we would be able to do debugging work in both platforms.

but for now, the goal is to just get this out the door and only have analytics for the production site.